packages/forma-36-react-components/README.md

```diff
@@ -101,7 +101,7 @@ We follow the principle that a component should only be responsible for its own
 We recommend the following naming convention for PropTypes to make them as clear as possible:
 
 - Number - use a prefix or suffix to imply that the prop accepts a number. E.g. `numItems`, `itemCount`, `itemIndex`
-- Boolean - use the prefix 'is'/'can'/'has'. E.g. `isVisible`, `canExpand`, `hasImage`
+- Boolean - use the prefix 'is'/'can'/'has'/'will'. E.g. `isVisible`, `canExpand`, `hasImage`
```

Comment:
was the `will` agreed upon somewhere?

Reply:
No, it wasn't. I thought it was left out by accident. What do you suggest for `willBlurOnEsc` ? `canBlurOnEsc` ?

Reply:
I haven't seen a use case like that here yet, but yes, I think I'd rather go with `canBlurOnEsc` than introduce another prefix.
Happy to be convinced otherwise as to potential future use cases though ☮️

Reply:
Mmmmh, I personally don't care about the decision. To avoid bikeshedding and not question a decision which was already made, I'm gonna take `canBlurOnEsc` :)